sdk/cognitivelanguage/ai-language-text/test/public/utils/customTestHelpter.ts

```diff
@@ -101,11 +107,30 @@ export async function createCustomTestProject(
       files = "LoanAgreements";
       break;
   }
-  const modelLabel = "projectModel";
-  await uploadDocuments(files);
-  logger.info("Finish uploading documents to storage");
 
-  const options: CreateProjectOptions = {
+  dir = path.join(pathName, files);
```

Comment:
Could we refactor this to be something the function returns instead in addition to the assets and files in one object? Global state can make it hard to reason about the code and it doesn't look to be necessary in this case. I recommend to always try to think of ways to use const variables instead of let ones.

Reply:
Thank you for the feedback. I did not pay attention to that and will update the functions accordingly